Makefile

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Why did you remove the Makefile?

Please look carefully at your diff and try to refrain from changing something if there is no strong motivation behind it. This will make the review process quicker and more pleasant for everyone involved.